include/zephyr/mgmt/ec_host_cmd/backend.h

```diff
@@ -21,98 +21,111 @@
 extern "C" {
 #endif
 
+struct ec_host_cmd_backend {
+	/** API provided by the backed.
+	 */
+	const struct ec_host_cmd_backend_api *api;
+	/** Context for the backed.
+	 */
+	void *ctx;
+};
+
 /**
- * @brief Host Command Backend API
- * @defgroup ec_host_cmd_backend Host Command Backend API
+ * @brief EC Host Command Interface
+ * @defgroup ec_host_cmd_interface EC Host Command Interface
  * @ingroup io_interfaces
  * @{
  */
 
 /**
- * @brief Context for host command backend and framework to pass rx data
+ * @brief Context for host command backend and handler to pass rx data
  */
 struct ec_host_cmd_rx_ctx {
-	/** Buffer written to by device (when dev_owns) and read from by
-	 *  command framework and handler (when handler_owns). Buffer is owned
-	 *  by devices and lives as long as device is valid. Device will never
-	 *  read from this buffer (for security reasons).
+	/** Buffer to hold received data. The buffer is provided by the handler if
```

Comment:
nit: Can't quite find a codingstyle entry but other code seems to be aligning multi line comments by leaving the first line with `/**` and nothing else, should this code stick to that convention as well?